src/main/java/solana/Solana.java

```diff
@@ -0,0 +1,382 @@
+                    false, false);
+            break;
+        case "[T][X]":
+            todoCommand(taskAsArray[DESCRIPTION_INDEX], false, true);
+            break;
+        case "[D][":
+            deadlineCommand(taskAsArray[DESCRIPTION_INDEX].substring(STARTING_DESCRIPTION_INDEX),
+                    false, false);
+            break;
+        case "[D][X]":
+            deadlineCommand(taskAsArray[DESCRIPTION_INDEX], false, true);
+            break;
+        case "[E][":
+            eventCommand(taskAsArray[DESCRIPTION_INDEX].substring(STARTING_DESCRIPTION_INDEX),
+                    false, false);
+            break;
+        case "[E][X]":
+            eventCommand(taskAsArray[DESCRIPTION_INDEX], false, true);
+            break;
+        default:
+            System.out.println("Unable to identify task type!");
+        }
+    }
+
+    public static void readFromFile(File savedTasks) throws FileNotFoundException {
+        Scanner in = new Scanner(savedTasks);
+        while (in.hasNext()) {
+            String taskAsString = in.nextLine().substring(TASK_INDEX);
+            String[] taskAsArray = taskAsString.split(" ", SPLIT_LIMIT);
+            parseInputFromFile(taskAsArray);
+        }
+    }
+
+    public static void loadTasks() {
+        File savedTasks = new File(FILE_PATH);
+        if (savedTasks.exists()) {
+            try {
+                readFromFile(savedTasks);
+            } catch (FileNotFoundException e) {
+                System.out.println("savedTasks.txt not found!");
+            }
+        }
+    }
+
+    public static void writeToFile() throws IOException {
+        FileWriter fw = new FileWriter(FILE_PATH);
+        int listNumber = STARTING_LIST_NUMBER;
+
+        for (Task task : tasks) {
+            fw.write(listNumber + ". ");
+            fw.write(task + System.lineSeparator());
+            listNumber++;
+        }
+        fw.close();
+    }
+
+    public static void saveTasks() {
+        File dataFolder = new File(FOLDER_PATH);
+        if (!dataFolder.exists()) {
+            boolean isSuccessful = dataFolder.mkdir();
+            if (!isSuccessful) {
+                System.out.println("Create data folder failed!");
+                return;
+            }
+        }
+
+        File savedTasks = new File(FILE_PATH);
+        try {
+            savedTasks.createNewFile();
+        } catch (IOException e) {
+            System.out.println("Create savedTasks.txt failed!");
+            return;
+        }
+
+        try {
+            writeToFile();
+        } catch (IOException e) {
+            System.out.println("Writing to savedTasks.txt failed!");
+        }
+    }
+
+    public static void checkDescription(String[] parsedInput) throws SolanaException {
+        if (parsedInput.length == 1) {
+            switch (parsedInput[COMMAND_INDEX]) {
+            case "bye":
+                //Fallthrough
```

Comment:
**Coding standard violation:** Add a whitespace character between `//` and `Fallthrough`

Reply:
Thanks for spotting that